tern/classes/image_layer.py

```diff
@@ -327,3 +327,10 @@ def add_files(self):
                 file_data.set_checksum('sha256', attrs_tuple[2])
                 file_data.extattrs = attrs_tuple[0]
                 self.add_file(file_data)
+
+    def get_layer_workdir(self):
+        match = re.search(r"\bWORKDIR\ (\/\w+)+\b", self.created_by)
+        if match:
+            return match.group().split()[1]
+        else:
+            return False
```

Comment:
Instead of returning a value if there is a match and False if there is not a match, I would return `None` here instead if there is no match.